src/Microsoft.DotNet.Build.Tasks/PackageFiles/versioning.targets

```diff
@@ -161,8 +161,15 @@
       <NativeVersionLines Condition="'$(Configuration)'=='Debug'" Include="#define VER_DEBUG                   VS_FF_DEBUG" />
       <NativeVersionLines Condition="'$(Configuration)'!='Debug'" Include="#define VER_DEBUG                   0" />
       <NativeVersionLines Include="#endif" />
+      <NativeVersionFile Include="$(NativeVersionHeaderFile)" />
```

Comment:
Today it is being set when we call msbuild in [corefx build.cmd](https://github.com/dotnet/corefx/blob/master/build.cmd#L78) and [coreclr build.cmd](https://github.com/dotnet/coreclr/blob/master/build.cmd#L297). But we could change that to use the ones in the property group instead.